6 дз/6 дз/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewLanguage
+{
+    class Program
+    {
+
+        public string Leet(string text)
+        {
```

Comment:
Правильно будет перед трансляцией в Leet перевести текст в нижний регистр в начале метода, а не полагаться, что это будет сделано при вызове метода.
```suggestion
        public string Leet(string text)
        {
            text = text.ToLower();
```